lambda/docker-runner.yaml

```diff
@@ -0,0 +1,111 @@
+# This template depends on having an AWS EFS in place which can be deployed with
+# https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/EFS/efs.yaml
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: S3Objects
+Description: >-
+  Docker runner product
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: Container Image
+        Parameters:
+          - ImageUri
+          - EntryPoint
+          - Command
+          - WorkingDirectory
+          - MemorySize
+      - Label:
+          default: Synapse
+        Parameters:
+          - SynapseUser
+          - SynapsePassword
```

Comment:
We should not call this a password if we want people to use other types of secrets... which I think is the case.

Reply:
will replace with a PAT

Reply:
replaced with a synapse personal access token `SynapsePAT`